test/spec_webrick.rb

```diff
@@ -1,11 +1,13 @@
+require 'minitest/bacon'
 require 'rack/mock'
 require File.expand_path('../testrequest', __FILE__)
 
 Thread.abort_on_exception = true
 
 describe Rack::Handler::WEBrick do
-  extend TestRequest::Helpers
+  include TestRequest::Helpers
 
+  before do
```

Comment:
The original tests only spin up webrick once.  This change makes webrick start on every test and causes `Errno::ECONNREFUSED` exceptions.  Try `ruby -I lib:test test/spec_webrick.rb`.